test/integration/QuickstartTest.kt

```diff
@@ -42,13 +49,13 @@ class QuickstartTest: IntegrationTest() {
 
     @Test
     fun Quickstart() {
-        studioTest(composeRule) {
-            connectToTypeDB(composeRule, DB_ADDRESS)
+        studioTestWithRunner(composeRule) { address ->
```

Comment:
We get the address passed to us by the runner as we cannot determine what the address will be at compile time, so when we need a runner in a test we have the address provided to us and we consume it in this manner.

The address contains the host and the port.